test/disabled/test-sendfd.js

```diff
@@ -127,9 +126,9 @@ var srv = net.createServer(function(s) {
 srv.listen(SOCK_PATH);
 
 // Spawn a child running test/fixtures/recvfd.js
-var cp = child_process.spawn(process.argv[0],
-                             [path.join(common.fixturesDir, 'recvfd.js'),
-                              SOCK_PATH]);
+var cp = child_process.spawn(process.argv[0], [path.join(common.fixturesDir, 'recvfd.js'),
```

Comment:
I did, and it failed first time
I changed, it passed
Not sure how this happened
Fixing now, sorry :-)